### DISCUSSION TITLE
Fundamental local Git not working.

### DISCUSSION THREAD

**Discussion by Wm-M-Thompson** [community/community#129749] [Programming Help]
### Select Topic Area

Question

### Body

For reasons beyond the scope of this discussion thread, I have found the need to install git locally on my work windows 10 computer.  But from the command line, the basic commands are not recognized.  For example:

`git config`

gets the response:

```
'git' is not recognized as an internal or external command,
operable program or batch file.

```

Can anyone please offer suggestions or next steps I should try?

**Comment by saeedhosan** [comment]
## If you are already installed Git on your windows you should add  Git to PATH:

  - Open the Start menu, search for "Environment Variables," and select "Edit the system environment variables."
  - In the System Properties window, click on the "Environment Variables" button.
  - In the Environment Variables window, find the "Path" variable in the "System variables" section,
  -  and click "Edit."In the Edit Environment Variable window, 
  - click "New" and add the path to your Git bin directory (e.g., C:\Program Files\Git\bin).
  - Click "New" again and add the path to your Git cmd directory (e.g., C:\Program Files\Git\cmd).
  - Click "OK" to close all windows.
  
### note 
the git path  C:\Program Files\Git\bin and  C:\Program Files\Git\cmd  can be different on your system please verify the path by explore files 

#### I hope this method will solve the  git command issues
